src/pages/ReportDetailsPage.tsx

```diff
@@ -196,10 +196,27 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
             });
         }
 
+        if ((isGroupChat || (isChatRoom && ReportUtils.canLeaveChat(report, policy ?? null))) && !isThread) {
+            items.push({
+                key: CONST.REPORT_DETAILS_MENU_ITEM.LEAVE_ROOM,
+                translationKey: 'common.leave',
+                icon: Expensicons.Exit,
+                isAnonymousAction: true,
+                shouldShowRightIcon: false,
```

Comment:
Should add shouldShowRightIcon: true to other item

Reply:
yeah, the current change work well, but I think add shouldShowRightIcon: true to other item will be clearer